### PR TITLE
Align configuration classes with Home Assistant standards

### PR DESCRIPTION
**Pull request by philippwaller**
This PR refines the configuration classes to better adhere to Home Assistant's coding standards and best practices.

### CODE REVIEW

File: custom_components/mbapi2020/switch.py [hunk 1]
```diff
@@ -6,159 +6,77 @@
...
+    attributes: list[str] | None = None
+    is_on_fn: Callable[[MercedesMESwitch], Callable[[], Coroutine[Any, Any, bool]]]
+    turn_on_fn: Callable[[MercedesMESwitch], Callable[[], Coroutine[Any, Any, None]]]
+    turn_off_fn: Callable[[MercedesMESwitch], Callable[[], Coroutine[Any, Any, None]]]
+    check_capability_fn: Callable[[Car], Callable[[], Coroutine[Any, Any, bool]]]
 
+SWITCH_CONFIGS: list[MercedesMeSwitchEntityDescription] = [
+    MercedesMeSwitchEntityDescription(
+        key="precond",
+        translation_key="Pre-entry climate control",
```
Comment (ReneNulschDE) [inline custom_components/mbapi2020/switch.py:45]: Translation key does not follow the rule r"^(?!.+[_-]{2})(?![_-])[a-z0-9-_]+(?<![_-])$" . You can remove this here as the key is has the correct format.

**Review by ReneNulschDE — CHANGES_REQUESTED** [review]
Thx - could you check the last comments please.

**Comment by ReneNulschDE** [comment]
I will implement the needed changes to get the release out. Thx!

**Comment by philippwaller** [comment]
Sorry, I was an roadtrip. Are you already working on it?

**Comment by ReneNulschDE** [comment]
No problem. I like your contributions and all the needed stuff is fixed. Just have a problem with the not working translation. Need to dive deeper tomorrow. Thanks again!

**Comment by philippwaller** [comment]
It’s strange that the translations aren’t being applied. I tested it on Friday morning and was quite sure that at least the German translations were showing up. I looked into it again last night but couldn’t spot the issue at first glance. 🤔

**Comment by ReneNulschDE** [comment]
OK - the sensor/switch title translation is not working at all. It showed the German value because your HA system language is DE. I will remove the german translation for now to make sure that the entities are created with english naming. I'll merge the PR with a few other changes and then work on the translation bug.

**Review by ReneNulschDE — APPROVED** [review]

**Pull request merged**